asminc/atari5200.inc

```diff
@@ -101,6 +111,14 @@ ANTIC     = $D400  ; ANTIC area
 POKEY     = $E800  ; POKEY area
 .include  "atari_pokey.inc"
 
+;-------------------------------------------------------------------------
+; conio color defines
+;-------------------------------------------------------------------------
+
+COLOR_WHITE      = 0
+COLOR_LIGHTRED   = 1
+COLOR_LIGHTGREEN = 2
```

Comment:
I asked for
* COLOR_BLACK
* COLOR_WHITE
* COLOR_RED
* COLOR_GREEN

Having COLOR_LIGHTRED and COLOR_LIGHTGREEN doesn't help in writing cross target code, which this was all about.

I presume you did it that way because the colors you use are named that way in _gtia.h. But that isn't a good reason from my pov.

Rather I question the names in _gtia.h. The color that "makes most sense to be used" needs to have the "primary" name. And the secondary color needs to have the secondary name. So if the brighter red and green are to be considered the major colors they need to be named COLOR_RED and COLOR_GREEN - and the other two can e.g. be named COLOR_DARKRED and COLOR_DARKGREEN (or any other name you like).

Anyhow, whatever approach you choose (different names, different colors, name aliases, ...) the result should have COLOR_RED and COLOR_GREEN defined.